src/llama.cpp

```diff
@@ -5152,10 +5152,10 @@ static void llm_load_vocab(
         if (gen_name.find("code") != std::string::npos) {
             if (model.arch == LLM_ARCH_LLAMA
               && 32010 < vocab.id_to_token.size()
-              && vocab.id_to_token[32007].text == "<PRE>"
-              && vocab.id_to_token[32008].text == "<SUF>"
-              && vocab.id_to_token[32009].text == "<MID>"
-              && vocab.id_to_token[32010].text == "<EOT>") {
+              && vocab.id_to_token[32007].text.ends_with("<PRE>")
+              && vocab.id_to_token[32008].text.ends_with("<SUF>")
+              && vocab.id_to_token[32009].text.ends_with("<MID>")
+              && vocab.id_to_token[32010].text.ends_with("<EOT>")) {
```

Comment:
`ends_with` is C++20, at the moment we target C++11.

Reply:
Yep, sorry, that's what I get for blindly submitting :(

I'll fix.